### PR TITLE
Add iptables INPUT rules for network policy to host networked pods

### PR DESCRIPTION
**Pull request by bazuchan**
Add iptables INPUT rules for enforcing network policy to host networked pods. Implementation of variant 2 from #616.

### CODE REVIEW

File: pkg/controllers/netpol/network_policy_controller.go [hunk 1]
```diff
@@ -684,6 +706,21 @@ func (npc *NetworkPolicyController) syncPodFirewallChains(version string) (map[s
 			}
 		}
 
+		if pod.ip == npc.nodeIP.String() {
+			comment := "rule to permit the traffic traffic to pods when source is the pod's local node"
```
Comment (lucasmundim) [inline pkg/controllers/netpol/network_policy_controller.go]: traffic traffic

Comment (bazuchan) [reply pkg/controllers/netpol/network_policy_controller.go]: fixed

**Comment by bazuchan** [comment]
Run travis again

**Comment by bazuchan** [comment]
Please keep it open for now, changes since is huge, I'll see what I can do.

**Comment by bazuchan** [comment]
Rebased

**Pull request closed without merging**